korman/properties/modifiers/gui.py

```diff
@@ -578,7 +601,7 @@ def _create_pots_nodes(self, clickable_object, nodes, linkingnode, age_name, clk
         responder.link_output(responder_state, "state_refs", "resp")
         responder.link_output(linkingnode, "keyref", "respOneShot")
 
-    def _create_moul_nodes(self, clickable_object, nodes, linkingnode, age_name, clk_region):
+    def _create_moul_nodes(self, clickable_object, nodes, linkingnode, age_name, clk_region, share_rgn):
```

Comment:
I think I'd prefer for this one to stay `share_region` to match `clk_region` for consistency.

Reply:
Fair enough. This won't conflict with the share region node stuff below that way, will it?

Reply:
You'll want to rename the share region node variable to something else

Reply:
Renamed to `share_msb_region` instead.